server/controllers/article_controller.js

```diff
@@ -50,18 +50,10 @@ export const addArticleAnnotation = (articleId, annotationId) => {
 // If user is null, return public annotations.
 // Returns a promise.
 
-export const getArticleAnnotations = (user, uri, topLevelOnly) => {
-  const conditions = {};
```

Comment:
we never used these conditions, not totally sure how they were supposed to be used? anyways, for now I got rid of them because it was superfulous.